server/plugin.go

```diff
@@ -193,7 +193,7 @@ func (p *Plugin) OnConfigurationChange() error {
 	}
 
 	encryptionKey := ec.EncryptionKey
-	if encryptionKey == "" {
+	if encryptionKey == "" && ec.AdminAPIToken != "" {
```

Comment:
Should have separate errors for these

Reply:
encryption key is only needed when admin API token is present